medusa/server/web/home/handler.py

```diff
@@ -1131,281 +1131,21 @@ def check_show_for_language(series_obj, language):
         if language in indexer.config['valid_languages']:
             return True
 
-    def editShow(self, indexername=None, seriesid=None, location=None, allowed_qualities=None, preferred_qualities=None,
-                 exceptions_list=None, season_folders=None, paused=None, directCall=False,
-                 air_by_date=None, sports=None, dvd_order=None, indexer_lang=None,
-                 subtitles=None, rls_ignore_words=None, rls_require_words=None,
-                 anime=None, blacklist=None, whitelist=None, scene=None,
-                 defaultEpStatus=None, quality_preset=None):
+    def editShow(self, indexername=None, seriesid=None):
+        t = PageTemplate(rh=self, filename='index.mako')
```

Comment:
If you want to remove this code from here, move it to `massEdit`

Reminding you this: #4075
> * [x]  Restore POST functionally to editShow because [**massEditSubmit still uses it**](https://github.com/pymedusa/Medusa/blob/130cfca37762342537d7d19720e42635a2fbe7b9/medusa/server/web/manage/handler.py#L681-L688)

and this commit: https://github.com/pymedusa/Medusa/pull/4075/commits/2400d77f78082f7101000501e4de505f3799b89c

Reply:
Tnx for the reminder.
My main focus now was to get the page ~~working~~ loading. So that's working.
Next will be getting the saving working. But I need to know how we're going to update store/api.

Reply:
I've choosen to create a new method `massEditShow` and move it to there.
Felt a little bit less messy for now.